src/main/java/org/broadinstitute/hellbender/utils/fragments/FragmentUtils.java

```diff
@@ -78,13 +78,13 @@ public static void adjustQualsOfOverlappingPairedFragments(final Pair<GATKRead,
             final byte firstReadBase = firstReadBases[firstReadIndex];
             final byte secondReadBase = secondReadBases[secondReadIndex];
 
-            if (firstReadBase == secondReadBase) {
```

Comment:
This seems like a very sensible change and this qual adjustment would seem to be an artifact of the fact that we very rarely set this flag that this has persisted like this. I would recommend adding a test to `FragmentUtilsUnitTest` in the same vein of `testAdjustingTwoReads()` that asserts that mismatching bases get adjusted accordingly.